src/pages/back-to-office/cat-c/back-to-office.cat-c.page.ts

```diff
@@ -0,0 +1,83 @@
+import { Component } from '@angular/core';
+import { IonicPage, NavController, NavParams, Platform } from 'ionic-angular';
+import { PracticeableBasePageComponent } from '../../../shared/classes/practiceable-base-page';
+import { AuthenticationProvider } from '../../../providers/authentication/authentication';
+import { Store, select } from '@ngrx/store';
+import { StoreModel } from '../../../shared/models/store.model';
+import { BackToOfficeViewDidEnter, DeferWriteUp } from '../back-to-office.actions';
+import { DeviceProvider } from '../../../providers/device/device';
+import { ScreenOrientation } from '@ionic-native/screen-orientation';
+import { Insomnia } from '@ionic-native/insomnia';
+import { JOURNAL_PAGE } from '../../page-names.constants';
+import { Observable } from 'rxjs/Observable';
+import { getTests } from '../../../modules/tests/tests.reducer';
+import { getCurrentTest } from '../../../modules/tests/tests.selector';
+import { getRekeyIndicator } from '../../../modules/tests/rekey/rekey.reducer';
+import { isRekey } from '../../../modules/tests/rekey/rekey.selector';
+import { CAT_C } from '../../../pages/page-names.constants';
+
+interface BackToOfficePageState {
+  isRekey$: Observable<boolean>;
+}
+
+@IonicPage()
+@Component({
+  selector: '.back-to-office-cat-c-page',
+  templateUrl: 'back-to-office.cat-c.page.html',
+})
+export class BackToOfficeCatCPage extends PracticeableBasePageComponent {
```

Comment:
Is there a practice mode for Cat C, or should Cat C always extend `BasePageComponent`?

Reply:
Nice spot! Should just extend BasePageComponent

Reply:
Also changed category B+E